pkg/injection/namespace/initgeneration/initgeneration.go

```diff
@@ -9,8 +9,9 @@ import (
 	"github.com/Dynatrace/dynatrace-operator/pkg/consts"
 	"github.com/Dynatrace/dynatrace-operator/pkg/injection/namespace/mapper"
 	"github.com/Dynatrace/dynatrace-operator/pkg/injection/startup"
-	"github.com/Dynatrace/dynatrace-operator/pkg/util/kubeobjects"
+	k8sobjectsecret "github.com/Dynatrace/dynatrace-operator/pkg/util/kubeobjects/secret"
 	"github.com/Dynatrace/dynatrace-operator/pkg/util/kubesystem"
+	labels2 "github.com/Dynatrace/dynatrace-operator/pkg/util/labels"
```

Comment:
```suggestion
	utillabels "github.com/Dynatrace/dynatrace-operator/pkg/util/labels"
```

Reply:
done in https://github.com/Dynatrace/dynatrace-operator/pull/2336/commits/b58b302251d3905a66492b1b0993d5ee3d7f2669